internal/bminventory/inventory.go

```diff
@@ -113,7 +113,7 @@ type OCPClusterAPI interface {
 type InstallerInternals interface {
 	RegisterClusterInternal(ctx context.Context, kubeKey *types.NamespacedName, params installer.RegisterClusterParams) (*common.Cluster, error)
 	GetClusterInternal(ctx context.Context, params installer.GetClusterParams) (*common.Cluster, error)
-	UpdateClusterInternal(ctx context.Context, params installer.UpdateClusterParams) (*common.Cluster, error)
+	UpdateClusterInternal(ctx context.Context, params installer.UpdateClusterParams, interactive bool) (*common.Cluster, error)
```

Comment:
suggesting to define a type for this boolean, something like https://github.com/openshift/assisted-service/blob/master/internal/common/db.go#L73-L85

Reply:
What's the benefit?

Reply:
That it's clearer when you see the code .
instead of having a function call like `update(..., true` it will be `update(..., Interactive` ``update(..., NoneInteractive`

Reply:
see for example the where `GetClusterFromDB` is used.

Reply:
Why not just define const? Why type?

Reply:
Changed to new API call without the flag - UpdateClusterNonInteractive